File: Hash/3.py
def solution(clothes):
    answer = 1
    dic = {}
    for var in clothes:
        if not var[1] in dic:
            dic[var[1]] = 1
        else:
            dic[var[1]] += 1
            
    for val in dic.values():
        answer *= (val+1)
    return answer-1

File: Sort/3.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<int> citations) {
    int answer = 0;
    sort(citations.begin(), citations.end(), greater<int>());
    
    while (answer < citations.size() && citations[answer] >= answer+1) 
        answer++;

    return answer;
}